src/main/java/mate/academy/service/AuthenticationService.java

```diff
@@ -11,6 +13,15 @@ public class AuthenticationService {
      * Return false in any other cases.
      */
     public boolean login(String email, String password) {
-        return false;
+        User founded = UserService.findByEmail(email);
+        if (founded != null) {
+            if (founded.getPassword() == password) {
+                return true;
+            } else {
+                return false;
+            }
+        } else {
+            return false;
+        }
```

Comment:
Let's simplify the code and write it in a single line

Reply:
try to use only one return statement in your solution

Reply:
Now the Login method uses only one return keyword